airbyte-local-cli-nodejs/src/command.ts

```diff
@@ -0,0 +1,286 @@
+    .addOption(
+      new Option(
+        '--src-only',
+        `Only run the Airbyte source and write output in stdout. Use '--src-output-file' instead to write to a file`,
+      ).conflicts('srcOutputFile'),
+    )
+    .option('--src-output-file <path>', 'Write source output as a file (requires a destination)')
+    .option('--dst-only <file>', 'Use a file for destination input instead of a source')
+    .option('--dst-use-host-network', 'Use the host network when running the Airbyte destination')
+    .option('--log-level <level>', 'Set level of source and destination loggers', 'info')
+    .option('--raw-messages', 'Output raw Airbyte messages')
+    .option('--connection-name <name>', 'Connection name used in various places')
+    .option('--keep-containers', 'Do not remove source and destination containers after they exit')
+
+    // Options: Cli settings
+    .option('--debug', 'Enable debug logging')
+
+    // Additional check
+    .action((opts: any) => {
+      if (opts.debug) {
+        updateLogLevel(true);
+        cmd.setOptionValue('logLevel', 'debug');
+      }
+
+      // Check if Airbyte config is provided
+      if (!opts.configFile && !(opts.src || opts.dst) && !opts.wizard) {
+        cmd.error(
+          'Configuration options are missing. Please provide one of the following options: ' +
+            `'--config-file', '--src' and '--dst', or '--wizard' to configure the Airbyte connector.`,
+        );
+      }
+    });
+  return cmd;
+}
+
+// Parse the key-value pair of source and destination configurations
+function parseSrcAndDstConfig(argv: string[]) {
+  const srcConfig: Record<string, string> = {};
+  const dstConfig: Record<string, string> = {};
+
+  function handleNestedConfig(obj: Record<string, any>, keys: string[], value: string) {
+    let current = obj;
+    keys.forEach((key, index) => {
+      if (index === keys.length - 1) {
+        current[key] = value;
+      } else {
+        if (!current[key]) {
+          current[key] = {};
+        }
+        current = current[key];
+      }
+    });
+  }
+
+  argv.forEach((arg, index) => {
+    if (arg.startsWith('--src.')) {
+      const keys = arg.split('.').slice(1);
+      const value = argv[index + 1];
+      if (keys.length && value && !value.startsWith('--')) {
+        handleNestedConfig(srcConfig, keys, value);
+      }
+    } else if (arg.startsWith('--dst.')) {
+      const keys = arg.split('.').slice(1);
+      const value = argv[index + 1];
+      if (keys.length && value && !value.startsWith('--')) {
+        handleNestedConfig(dstConfig, keys, value);
+      }
+    }
+  });
+  logger.debug('Source Config:', srcConfig);
+  logger.debug('Destination Config:', dstConfig);
+  return {srcConfig, dstConfig};
+}
+
+// Convert the options to CliOptions type
+function convertToCliOptions(options: any) {
+  return {
+    ...options,
+    srcImage: options.src,
+    dstImage: options.dst,
```

Comment:
simply rename the image to new name. 
src/dst being the image flags are kinda a headache and ambiguous to the src.* / dst.* options